packages/api/portfolioDrafts/application/getApplicationStep.ts

```diff
@@ -18,14 +27,16 @@ export const NO_SUCH_PORTFOLIO_DRAFT = new OtherErrorResponse(
  *
  * @param event - The GET request from API Gateway
  */
-export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> {
-  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
-  if (!isPathParameterPresent(portfolioDraftId)) {
-    return PATH_PARAMETER_REQUIRED_BUT_MISSING;
-  }
-  if (!isValidUuidV4(portfolioDraftId)) {
-    return NO_SUCH_PORTFOLIO_DRAFT;
+export async function baseHandler(
+  event: ApiGatewayEventParsed<ApplicationStep>,
+  context?: Context
+): Promise<APIGatewayProxyResult> {
+  const setupResult = shapeValidationForPostRequest<ApplicationStep>(event);
```

Comment:
This is a GET not a POST, we may want to rename it to something like: `requestShapeValidation`

Reply:
I think `requestShapeValidation` is good. I know you are changing it on your branch, so I won't do it here.